package.json

```diff
@@ -42,7 +37,7 @@
     "prettier": "^2.4.1",
     "react": ">=16.0 <18.0",
     "react-dom": ">=16.0 <18.0",
-    "replicache": ">= 10.0.0",
+    "replicache": "10.0.0",
```

Comment:
I would expect this to be `^10.0.0`

Reply:
Well here's the way I think of it -- we're not actually relying on a particular version . This code is basically do the same thing as the Go pattern of having consumers declare the interfaces for their dependencies, except that in this case the interface is so complex (ReadTransaction) that I'm borrowing the definition in Replicache 10. I could actually pick any version of Replicache that has an interface that is compatible with what useSubscribe is trying to do. I sort of arbitrarily picked 10.

Reply:
I agree.

I was just commenting on locking the version like this. Generally you'd want to use semver.